src/jimgw/run/library/class_definitions.py

```diff
@@ -0,0 +1,12 @@
+import enum
```

Comment:
This file aims to create an enumeration of available definition so the user/a run time can execute a run through the interface: python execute --run_type X --config file